content/docs/command-reference/checkout.md

```diff
@@ -75,15 +75,16 @@ the [pipeline](/doc/command-reference/dag) must be reproduced (using
 - `--summary` - display a short summary of the changes done by this command in
   the workspace, instead of a full list of changes.
 
+- `-d`, `--with-deps` - only meaningful when specifying `targets`. This
+  determines files to update by resolving all dependencies of the target stages
```

Comment:
List this option first (as in most other refs) since it's more broad: applies to any target (`-R` only applies to dir targets).

Reply:
Also, the wording is standardized across refs.